src/main/java/org/sefgobal/dataholder/api/YoutubeAPI.java

```diff
@@ -33,6 +34,11 @@ public void getOneLive(@RequestParam(required = false) String nextPageToken, Htt
             url = url + "&pageToken=" + nextPageToken;
         }
 
+        // Check if the user has sent a specific number of videos
```

Comment:
Great! It's nice to see these comments. It makes this code more readable. 😍

- [ ] Shall we change it as follows?
  ` // Check if the user has sent a custom value for maxResults`

- [ ] Also, could you please remove the extra white lines? 🤗
    line 41, 36, 31. 

- [ ] If it is possible could you please add a comment to the line 41?